tommath_private.h

```diff
@@ -202,21 +218,25 @@ mp_err func_name (mp_int * a, type b)                    \
 }
 
 /* deprecated functions */
-MP_DEPRECATED(s_mp_invmod_fast) mp_err fast_mp_invmod(const mp_int *a, const mp_int *b, mp_int *c);
-MP_DEPRECATED(s_mp_montgomery_reduce_fast) mp_err fast_mp_montgomery_reduce(mp_int *x, const mp_int *n, mp_digit rho);
-MP_DEPRECATED(s_mp_mul_digs_fast) mp_err fast_s_mp_mul_digs(const mp_int *a, const mp_int *b, mp_int *c, int digs);
-MP_DEPRECATED(s_mp_mul_high_digs_fast) mp_err fast_s_mp_mul_high_digs(const mp_int *a, const mp_int *b, mp_int *c,
+MP_PRIVATE MP_DEPRECATED(s_mp_invmod_fast) mp_err fast_mp_invmod(const mp_int *a, const mp_int *b, mp_int *c);
```

Comment:
Everything still stays public, since MP_PRIVATE_SYMBOLS must be defined to enable MP_PRIVATE. This is an opt-in feature. I have no idea how to do that differently since it will break the ABI. My recommendation would be to just make MP_PRIVATE effective when we remove all the deprecated stuff since at this point we are already breaking the ABI.

Reply:
We could however also remove MP_PRIVATE from the deprecated functions since it would not make a difference in the end. Is this what you mean?

Reply:
Yeah, they will be removed anyways as soon as we enable MP_PRIVATE

Reply:
Ok, I will remove MP_PRIVATE from the deprecated ones then. MP_DEPRECATED already signals that they are going to be removed.

Reply:
and activating MP_PRIVATE_SYMBOLS now would then produce an incompatible but finally correct version of ltm regarding private/public of the API :-)

Reply:
Well this is the least of the problems, MP_PRIVATE_SYMBOLS would already produce an incompatible version since it also hides the other private functions.

Reply:
> since it also hides the other private functions

that's what I meant with "finally correct version"

Reply:
Ok. I think I misunderstood you but it's ok now. Let's discuss more important things - may I ask you to comment on #262, #278, #279 ;)